cmd/aries-agent-mobile/pkg/wrappers/command/aries_test.go

```diff
@@ -64,3 +64,16 @@ func TestAries_GetDIDExchangeController(t *testing.T) {
 		require.NotNil(t, dec)
 	})
 }
+
+func TestAries_GetIssueCredentialController(t *testing.T) {
+	t.Run("test it creates an issue credential controller instance", func(t *testing.T) {
```

Comment:
nit: for the single test no need for `t.Run`.
The test name is too long. I would call it `Create controller `. 
The output will be clear enough. 
e.g `TestAries_GetIssueCredentialController/Create_controller`.

Reply:
fixed